From: chrysn <chrysn@fsfe.org>
Date: Mon, 8 Jun 2015 13:03:47 +0200
Subject: test_pretty_print: drop file name munging

forcibly joining the paths breaks the test suite when run out-of-tree
(as it is run when using debhelper)
---
 tests/test_pretty_print.py | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/tests/test_pretty_print.py b/tests/test_pretty_print.py
index 2e22feb..9e0cb50 100644
--- a/tests/test_pretty_print.py
+++ b/tests/test_pretty_print.py
@@ -380,9 +380,8 @@ def to_html(project_name, startdate, tests, enddate, sysinfo, sysid, imgcomparer
             raise TypeError('Unknown test type %r' % test.type)
 
     for mf in sorted(makefiles.keys()):
-        mfname = mf.strip().lstrip(os.path.sep)
-        text = open(os.path.join(builddir, mfname)).read()
-        templates.add('makefile_template', 'makefiles', name=mfname, text=text)
+        text = open(mf).read()
+        templates.add('makefile_template', 'makefiles', name=mf, text=text)
 
     text_tests = templates.get('text_tests')
     image_tests = templates.get('image_tests')
